src/core/enum_type.hpp

```diff
@@ -44,4 +44,16 @@
 		return static_cast<OtherEnumType>(static_cast<typename std::underlying_type<OtherEnumType>::type>(m1) + static_cast<typename std::underlying_type<EnumType>::type>(m2)); \
 	}
 
+/**
+ * Checks if a value in a bitset enum is set.
+ * @param x The value to check.
+ * @param y The flag to check.
+ * @return True if the flag is set, false else.
```

Comment:
```suggestion
 * @return True iff the flag is set.
```

As you suggested to me recently (because there's a typo anyway) 😉

Reply:
Arrrr copied from HasBit()